sdks/jreleaser-bitbucket-cloud-java-sdk/src/main/java/org/jreleaser/sdk/bitbucket/cloud/BitbucketCloudReleaserBuilderFactory.java

```diff
@@ -0,0 +1,21 @@
+package org.jreleaser.sdk.bitbucket.cloud;
+
+import org.jreleaser.model.spi.release.ReleaserBuilderFactory;
+import org.kordamp.jipsy.annotations.ServiceProviderFor;
+
+/**
+ * @author Hasnae Rehioui
+ * @since 1.6.0
+ */
+@ServiceProviderFor(ReleaserBuilderFactory.class)
+public class BitbucketCloudReleaserBuilderFactory implements ReleaserBuilderFactory<BitbucketCloudReleaser, BitbucketCloudReleaserBuilder> {
```

Comment:
I suspect this is documented somewhere , 
but I got here with trial and error by running `mvnDebug jreleaser:release -f pom.xml -e` on the following hello world fork https://bitbucket.org/viqueen/helloworld-java-jar/src/main/

I will have to switch back to "reading" mode and assess if there are any gaps between what's documented and what I ended up doing

Reply:
Err no, not really 😅 
This class is used by `Releasers` via ServiceLoader API to locate an appropriate Releaser instance.